spec/std/sprintf_spec.cr

```diff
@@ -0,0 +1,443 @@
+require "./spec_helper"
+require "../support/string"
+{% unless flag?(:win32) %}
+  require "big"
+{% end %}
+
+# rearrange parameters for `sprintf` and `IO#printf`
+private def fprintf(format, *args)
+  sprintf(format, *args)
+end
```

Comment:
Is this still needed? It says `rearrange parameters` but they're passed in the same order as received.

Reply:
@HertzDevil am I missing sth, or the `io` overload doesn't seem to be used?

Reply:
`assert_prints` needs a common method name.